### DISCUSSION TITLE
花12萬買M1 ultra是盤子嗎?

### DISCUSSION THREAD

**Discussion by PoooYen** [1101-GS4719-C/Python-Community-GS4719-C#441] [General]
在蘋果推出過全新M1、M1 Pro、M1 Max的高效能晶片以後，今年蘋果更把最高規格的M1 Max晶片合在一起，打造出最強大的M1 Ultra晶片，帶來前所未有性能的系統單晶片 (SoC)，為個人電腦晶片史上最多包含1140億個的電晶體。M1 Ultra擁有20核的CPU、最高64核的GPU及32核的神經元引擎，最高支援128GB RAM規格，對比一般16核CPU的桌機來說，就少用上100W的功耗，GPU更是比起現在熱門的獨立顯卡功耗，更是少了200W即能達到同樣的效果，因此也誕生了最強大的個人電腦Mac Studio，主攻擁有龐大創作需求的使用者，能以史無前例的流暢體驗來完成創作。

https://www.chinatimes.com/realtimenews/20220309000896-260412?chdtv

以個人觀點來看，M1的性能大家都知道是海放intel還有amd的，這次居然將兩個M1 MAX合併放在一台主機上，蘋果真的是太狠了，而且如有需要大量的繪圖及影音剪輯所省下來的時間會非常可觀，電費亦然(砸錢環保愛地球的部分)，有興趣的可以去看Joeman的開箱XD，不過還是要回歸到消費者身上是不是需求有到這麼頂，但我覺得在同樣的價格級距下真的是海放所有品牌，尤其現在光一張3080Ti 3090的顯卡動輒6~7萬 蘋果12萬幫你配到好，而且海放現在市面上所有主機，再給你滿滿優越的品牌光環能不香嗎(我不是業配)。 最後的最後.....好想當盤子但我還是買不起QQ

**Comment by HuanYuuuuuu** [comment]
我覺得除非真的有需要用到那麼好的性能的人(加上信仰)，因為感覺學生一般的intel core i5 i7 基本上就很夠用了。

**Reply by PoooYen** [reply]
但是intel再不努力一點為了就都是蘋果的市場了...目前我只覺得Windows的介面還有軟體相容性贏過apple其他都...

**Comment by ThomasMeng0515** [comment]
蘋果自行研發的晶片真的效能很頂，像iphone的A15晶片性能就幾乎沒有對手~

**Comment by lx3211** [comment]
iPad air5也配上M1的晶片還更便宜....
雖然是去年暑假買的ari4但還是有點微心痛

**Reply by PoooYen** [reply]
去年買M1 Pro現在有點後悔了....但還是早買早享受辣

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，蘋果的晶片真強。

**Comment by HsuChen88** [comment]
有錢的人不一定有這麼高的需求，但有需求的人可能買不起，只好花更多時間賺比較少的錢。:sob:

**Comment by handsomeniko** [comment]
看需求吧 蘋果介面的確做得好

**Comment by Rosancl** [comment]
看了一下自己錢包，嗯我還是i5就好了 ;)

**Comment by Sazmafuji** [comment]
去apple官網套一頂配的電腦會超過100萬，我真笑死

**Comment by jason79461385** [comment]
那些電腦都是給專業人士用的，假如製作一小段4K動畫能少個十分鐘，差別就很大了。

**Comment by alexthebigpp** [comment]
蘋果除了品牌走高檔路線外，她產品的品質、效能也是比windows系列還要好，如果說沒了相容性的問題，他的電腦應該市占率又會再提升不少。

**Comment by yenwei12138** [comment]
這麼高的價格，除非真的非常非常需要用到那麼好的效能，一般庶民應該都下不了手吧......

**Comment by yucheng1028** [comment]
我看了看荷包 恩 我還是看看就好
